apm-agent-plugins/apm-slf4j-plugin/src/test/java/co/elastic/apm/agent/slf4j/Slf4JMdcActivationListenerTest.java

```diff
@@ -134,7 +134,30 @@ void testWithNullContextClassLoader() throws Exception {
         });
         thread.setContextClassLoader(null);
         thread.start();
-        assertThat(result.get(1000, TimeUnit.MILLISECONDS).booleanValue()).isTrue();
+        assertThat(result.get().booleanValue()).isTrue();
+        assertMdcIsEmpty();
+        thread.join();
+        transaction.end();
+    }
+
+    @Test
+    void testWithNullApplicationClassLoaderFallbackPlatformCL() throws Exception {
+        when(loggingConfiguration.isLogCorrelationEnabled()).thenReturn(true);
+        Transaction transaction = tracer.startTransaction(TraceContext.asRoot(), null, null).withType("request").withName("test");
```

Comment:
What about a positive test for the fallback class loader (ie when the context class loader is properly set)?
It can be the same test, but without setting the Platform CL, then the the context CL would be System CL